### ISSUE TITLE
family_test.UT_IsSameFamily.test_04_roster_orphan raises Error

### ISSUE THREAD

**Issue by djfkahn** [djfkahn/MemberHubDirectoryTools#97]
```
WHERE:
	 test_04_roster_orphan (family_test.UT_IsSameFamily)
WHAT:
 Traceback (most recent call last):
  File "/Users/djfkahn/Desktop/DEV/MemberHubDirectoryTools/TEST/family_test.py", line 193, in test_04_roster_orphan
    self.assertFalse(this.IsSameFamily(that))
  File "/Users/djfkahn/Desktop/DEV/MemberHubDirectoryTools/TEST/family.py", line 99, in IsSameFamily
    if self.FindAdultInFamily(other_adult) != None:
  File "/Users/djfkahn/Desktop/DEV/MemberHubDirectoryTools/TEST/family.py", line 182, in FindAdultInFamily
    if to_find.IsSame(adult):
  File "/Users/djfkahn/Desktop/DEV/MemberHubDirectoryTools/TEST/person.py", line 36, in IsSame
    if self.family_relation[:5].lower() != other.family_relation[:5].lower():
TypeError: 'NoneType' object is not subscriptable
```

**Comment by djfkahn** [comment]
This issue was traced to Family.IsOrphan not returning True when all the members of the family's adults have first name = last name = "None".

**Comment by djfkahn** [comment]
addressed in commit 8089df7
